ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupDemoController.swift

```diff
@@ -0,0 +1,279 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+// MARK: - AvatarGroupDemoController
+
+class AvatarGroupDemoController: DemoController {
```

Comment:
Do you intend to change this to a table view in this PR?

Reply:
Yes, I realized I did not address that so it is being fixed